### DISCUSSION TITLE
手机上测速的所有结果都是无速度，电脑可以正常测速，我使用的是同一个测速地址

### DISCUSSION THREAD

**Discussion by woaiguyu1314** [XIU2/CloudflareSpeedTest#442] [提问] [answered]
手机上测速的所有结果都是无速度，电脑可以正常测速，我使用的是同一个测速地址

**Comment by charSLee013** [comment, marked as answer]
有几种情况你可以针对性分析下：

1. 电脑与手机不是同个网络环境，自然结果也就不同
2. 在同个网络环境下，但是因为每次测速的IP默认都是**随机选择**，自然测速的结果也不一样，这种情况下添加参数`-allip`解决
3. 手机与电脑连接网络的**物理方式不一样**，比如说手机是用WiFi，而电脑是用网线，一些性能羸弱的路由器WiFi性能并不是很好，导致测速失败
4. 手机**无线模块**问题，你可以在测速的时候后台播发直播或者其他等同的网络行为，看看会不会出现卡顿或者网络无响应情况，这说明其实手机已经跟路由器断开连接了

**Reply by woaiguyu1314** [reply]
那个我是同网络同参数，以前手机都有速度但是卸过一次后测出来就没速度，延迟有速度为0

**Reply by charSLee013** [reply]
> 延迟有速度为0

这是不是手机的clash或者其他代理工具没关？一般延迟测速都会在40ms以上或者更高

**Reply by woaiguyu1314** [reply]
好了测速地址有点问题
你好那个你知道测试延迟的话怎么才能把所有符合要求的结果输出出来呀

**Reply by XIU2** [reply]
@woaiguyu1314 如果你指的是将所有结果显示出来，那么可以加上 `-p 9999`，该参数控制的就是最后软件输出的结果数量（默认 10 个）。
